app/uk/gov/hmrc/minorentityidentification/models/CtutrValidationResult.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.minorentityidentification.models
+
+sealed trait CtutrValidationResult
+
+case object DetailsMatched extends CtutrValidationResult
+
+case object DetailsMismatched extends CtutrValidationResult
+
+case object DetailsNotFound extends CtutrValidationResult
```

Comment:
Not sure if it is producing a warning but usually if the file doesn't end with a new line we get a warning